### ISSUE TITLE
Untranslatable strings in the menu bar

### ISSUE THREAD

**Issue by gunchleoc** [LLK/scratch-gui#4251]
There are 2 strings in the Menu bar that still appear in English:

1. The default file name (`untitled-{number}`)
2. A button (`{symbol} See Project Page`)

My translation is at 100% reviewed and the latest translations have been pulled, so I expect there is some i18n markup missing or the strings aren't being picked up.

![untranslatables-menu-bar](https://user-images.githubusercontent.com/4095570/50682505-56e65400-100f-11e9-9f52-edbeb74de95b.PNG)

**Comment by Kenny2github** [comment]
Is the `gui.menuBar.seeProjectPage` string available on Transifex for you?

**Comment by apple502j** [comment]
In Japanese 2nd is fine.

**Comment by gunchleoc** [comment]
Yes, I see it in Transifex in the Scratch Editor project, in the interface resource. It has been translated and reviewed.

**Comment by chrisgarrity** [comment]
The translations for the launch were pulled on Dec 20, and the `see project page` change was very recent, so it may not have been translated/reviewed at that time. We're planning to include translations in the next update, barring unforeseen problems, your translations should be on the site by next week.

**Comment by chrisgarrity** [comment]
The 'Untitled' name is coming from the legacy system and is not translatable. When we move off the legacy system we should make sure that we can localize the default title. /cc @thisandagain
